### PR TITLE
Demonstrate the PDF generation for the blog

### PR DESCRIPTION
**Pull request by vlad-lubenskyi**
In this changeset, we add a simple example application for an upcoming tutorial in the blog.

This application demonstrates how:

1. To load the template page.
2. Fill it with real data.
3. Configure the PDF printer.
4. Print.

### CODE REVIEW

**Comment by anna-dolbina** [comment]
1. Please move this example to `csharp` subfolder, as it is written in C#.
2. The project name from the article (`PdfGeneration`) sounds better. We don't use `HowToXxx` as the project names.

**Comment by anna-dolbina** [comment]
We also have an existing example 
https://github.com/TeamDev-IP/DotNetBrowser-Examples/tree/master/csharp/console/Printing.WebPageToPdf
Consider merging the code to be able to reuse the existing one.

**Comment by vlad-lubenskyi** [comment]
@anna-dolbina PTAL

File: csharp/console/Printing.WebPageToPdf/Program.cs [hunk 1]
```diff
@@ -41,92 +37,84 @@ namespace Printing.WebPageToPdf
...
                 RenderingMode = RenderingMode.OffScreen,
-                GpuDisabled = true
+                LicenseKey = "your license key"
             }.Build();
 
-            using (IEngine engine = EngineFactory.Create(engineOptions))
-            {
-                using (IBrowser browser = engine.CreateBrowser())
-                {
-                    // #docfragment "PrintToPdf"
```
Comment (vlad-lubenskyi) [inline csharp/console/Printing.WebPageToPdf/Program.cs:60]: I see that I'm breaking the support portal here. I'll adjust it.

**Comment by anna-dolbina** [comment]
@vlad-lubenskyi please make sure that the "docfragment" markup is correct. The code snippet is used in the existing articles.

File: csharp/console/Printing.WebPageToPdf/Program.cs [hunk 2]
```diff
@@ -41,92 +37,90 @@ namespace Printing.WebPageToPdf
...
+            // #docfragment "PrintToPdf"
+
+            var whenPrintCompleted = ConfigurePrinting(browser);
+            browser.MainFrame.Print();
+            var resultPath = await whenPrintCompleted.Task;
+            Console.WriteLine($"PDF is generated: {resultPath}");
             Console.WriteLine("Press any key to terminate...");
             Console.ReadKey();
         }
+
```
Comment (anna-dolbina) [inline csharp/console/Printing.WebPageToPdf/Program.cs:64]: Extra empty line

File: csharp/console/Printing.WebPageToPdf/Program.cs [hunk 3]
```diff
@@ -41,92 +37,90 @@ namespace Printing.WebPageToPdf
...
+        private static TaskCompletionSource<string> ConfigurePrinting(IBrowser browser)
+        {
+            // Tell the browser to print automatically instead of showing the print preview.
+            browser.RequestPrintHandler = new Handler<RequestPrintParameters, RequestPrintResponse>(
+               p => RequestPrintResponse.Print()
+            );
+
+            TaskCompletionSource<string> whenCompleted = new();
+            // When the browser prints an HTML page.
```
Comment (anna-dolbina) [inline csharp/console/Printing.WebPageToPdf/Program.cs:74]:
```suggestion
            // Configure how the browser prints an HTML page.
```

**Comment by vlad-lubenskyi** [comment]
@anna-dolbina it renders OK. 

We know start creating a separate project purely for code samples for the documentation site. Once we try it with JxBrowser, we'll do the same for DotNetBrowser, and it will solve the problem.

**Review by anna-dolbina — APPROVED** [review]
We need to update the VB.NET example as well to make them identical. This will be done in a separate PR.

**Pull request merged**